packages/cdk/lib/constants.ts

```diff
@@ -9,102 +11,8 @@ export const USER_EMAIL_TAG_KEY = `${APP_NAME}-user-email`;
-- /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -m ec2 -a status | jq -r '.status' | grep -iw "running" || sleep 10 && /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c file:/opt/aws/amazon-cloudwatch-agent/etc/config.json
-- /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -m ec2 -a status | jq -r '.status' | grep -iw "running" || shutdown -P now
-
-# install aws-cli v2 and copy the static binary in an easy to find location for bind-mounts into containers
-- mkdir -p /opt/aws-cli/bin
-- curl -s "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip" && unzip -q /tmp/awscliv2.zip -d /tmp && /tmp/aws/install -b /usr/bin && cp -a -f $(dirname $(find /usr/local/aws-cli -name 'aws' -type f))/. /opt/aws-cli/bin/
-- command -v aws || sleep 5 | curl -s "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip" && unzip -q /tmp/awscliv2.zip -d /tmp && /tmp/aws/install -b /usr/bin && cp -a -f $(dirname $(find /usr/local/aws-cli -name 'aws' -type f))/. /opt/aws-cli/bin/
-- command -v aws || sleep 10 | curl -s "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip" && unzip -q /tmp/awscliv2.zip -d /tmp && /tmp/aws/install -b /usr/bin && cp -a -f $(dirname $(find /usr/local/aws-cli -name 'aws' -type f))/. /opt/aws-cli/bin/
-- command -v aws || shutdown -P now
-
-# set environment variables for provisioning
-- export ARTIFACTS_NAMESPACE=${APP_NAME}
-- export INSTALLED_ARTIFACTS_S3_ROOT_URL=$(aws ssm get-parameter --name /\${ARTIFACTS_NAMESPACE}/_common/installed-artifacts/s3-root-url --query 'Parameter.Value' --output text)
-
-# enable ecs spot instance draining
-- echo ECS_ENABLE_SPOT_INSTANCE_DRAINING=true >> /etc/ecs/ecs.config
-
-# pull docker images only if missing
-- echo ECS_IMAGE_PULL_BEHAVIOR=prefer-cached >> /etc/ecs/ecs.config
-
-# Setup ecs additions
-- cd /opt
-- aws s3 sync \${INSTALLED_ARTIFACTS_S3_ROOT_URL}/ecs-additions/ ./ecs-additions && chmod a+x /opt/ecs-additions/provision.sh  
-- test -f ./ecs-additions/fetch_and_run.sh || sleep 5 || aws s3 sync \${INSTALLED_ARTIFACTS_S3_ROOT_URL}/ecs-additions/ ./ecs-additions && chmod a+x /opt/ecs-additions/provision.sh    
-- test -f ./ecs-additions/fetch_and_run.sh || sleep 10 || aws s3 sync \${INSTALLED_ARTIFACTS_S3_ROOT_URL}/ecs-additions/ ./ecs-additions && chmod a+x /opt/ecs-additions/provision.sh    
-- test -f ./ecs-additions/fetch_and_run.sh || shutdown -P now
-- /opt/ecs-additions/provision.sh
+export const ENGINE_CROMWELL = "cromwell";
+export const ENGINE_MINIWDL = "miniwdl";
+export const ENGINE_NEXTFLOW = "nextflow";
 
-- echo "successfully initiated"
---==MYBOUNDARY==--
-`;
+export const wesAdapterSourcePath = path.resolve(path.join(__dirname, "./wes_adapter"));
```

Comment:
This doesn't seem right - isn't it stored outside of the `cdk` package now. I also don't see this being used anywhere

Reply:
Agreed, seems like a left over from my rebase.